library/platform_util.c

```diff
@@ -133,3 +134,14 @@ struct tm *mbedtls_platform_gmtime_r( const mbedtls_time_t *tt,
 #endif /* _WIN32 && !EFIX64 && !EFI32 */
 }
 #endif /* MBEDTLS_HAVE_TIME_DATE && MBEDTLS_PLATFORM_GMTIME_R_ALT */
+
+#if defined(MBEDTLS_CHECK_PARAMS) && defined(MBEDTLS_DEBUG_INVALID_PARAMS)
```

Comment:
What is the purpose of `MBEDTLS_DEBUG_INVALID_PARAMS`? It does not look like its documented anywhere.